tensorflow/tools/ci_build/gpu_build/parallel_gpu_execute.sh

```diff
@@ -67,8 +67,7 @@ for j in `seq 0 $((TF_TESTS_PER_GPU-1))`; do
       (
         # This export only works within the brackets, so it is isolated to one
         # single command.
-        export CUDA_VISIBLE_DEVICES=$i
-        export HIP_VISIBLE_DEVICES=$i
+        export ROCR_VISIBLE_DEVICES=$i
```

Comment:
this script is used by the `cuda` CI run too. 
we cannot remove the line that sets `CUDA_VISIBLE_DEVICES=$1` line

Reply:
Agreed, that's why I opened the following HIP#1660 PR to remove CUDA_VISIBLE_DEVICES support in HIP runtime. With that, we can keep the CUDA_VISIBLE_DEVICES env var unchanged here.

Reply:
I don't think it's a good idea to remove CUDA_VISIBLE_DEVICES support, at least not without adding info about ROCR_VISIBLE_DEVICES and/or HIP_VISIBLE_DEVICES someplace prominent in HIP documentation. We don't know who might be relying on it and it will break existing users.

A reasonable tradeoff might be to patch HIP to ignore CUDA_VISIBLE_DEVICES only if it sees  ROCR_VISIBLE_DEVICES (to prevent the devices from being filtered twice).

Reply:
@ekuznetsov139 , we won't remove the `CUDA_VISIBLE_DEVICES` option in this file. 
After the HIP PR#994 is available, I'll revert the change on this line.

Reply:
I mean that we don't want to remove the support of CUDA_VISIBLE_DEVICES from HIP (HIP PR#1660).